### DISCUSSION TITLE
Teams v3 - a few suggestions

### DISCUSSION THREAD

**Discussion by jesperordrup** [umami-software/umami#2569] [Ideas]
### Describe the feature or enhancement

I like the new 2.10 version shows team websites on dashboard. But there are some things that doesnt feel in place

Why teams? They feel very much in the way. How may times do people actually use the same team for multiple websites? Isnt there always someone unique to be added thus requiring a new team anyway?

Invite users directly to websites would be a very clear, direct easy way that I think would be more useful. 

Now you have teams owning websites. Consider giving access to websites by inviting teams and users directly. Reduce teams to be a grouping of users.

This would also allow users to have roles in relation to specific websites.

**Comment by franciscao633** [comment]
We don't believe having the granularity per website is a good thing, but we can disagree. The way teams is structured now you setup a team, setup the members/roles, and what those members are able to do with your team naturally flows down. No need to manage at the website level. We do appreciate the input, and will use your feedback in future iterations.

**Reply by jesperordrup** [reply]
Thanks i understand the current setup in which  you believe in. And there's probably a good reason for it. I just can't find it 😀

Sometimes these structures are there for a reason and sometimes they just happened to be made that way without a specific usecase in mind.

My suggestion does not eliminate teams. It's just a bit more flexible and afaik without any drawbacks

**Comment by gigor** [comment]
+1. Teams are very confusing and feel unnecessary. Additionally having to click Username > Switch Account > Team Name > Websites is very unintuitive. Why not just list all the websites one has access to on the homepage?
